pkg/webhook/admission/validator.go

```diff
@@ -0,0 +1,169 @@
+package admission
```

Comment:
This file will grow beyond reasonable size. I'd rename it to be specific for nic.
Then add another file for disk, affinity and so on.

Reply:
:+1: I already have my storage validation implemented this way on a branch.

Reply:
It looks like this: https://github.com/jakub-dzon/vm-import-operator/tree/storage_rules/pkg/webhook/admission. If that's OK, I will keep the split-related changes there.